modules/docs/mdx/versionsTable.tsx

```diff
@@ -69,6 +41,22 @@ function getVersionStatusIndicator(versionNumber: number | string) {
 }
 
 export const VersionTable = () => {
+  const [versions, setVersions] = React.useState<VersionType[]>([]);
+  const minVersion = 6;
+  const currentMajorVersion = Number(version?.split('.')[0]);
+  React.useEffect(() => {
+    let arr: VersionType[] = [];
+    for (let i = minVersion; i <= currentMajorVersion; i++) {
+      arr.push({
+        versionNumber: i,
+        versionUrl:
+          i === currentMajorVersion
+            ? 'https://canvas.workday.com/'
+            : `https://canvas.workday.com/v${i}/`,
+      });
+    }
+    setVersions(arr);
```

Comment:
@RayRedGoose  show me how to improve this. Basically mapping over the amount of versions we have to make an array of objects that have the number and version url